Makefile

```diff
@@ -9,6 +9,9 @@ SRC=$(shell find . -name '*.go')
 all: test
 
 deps:
+	go get github.com/jteeuwen/go-bindata/...
+	go get github.com/elazarl/go-bindata-assetfs/...
+	go generate
```

Comment:
does this need to be done before `go get` or could it be added to the eremetic block before `go build`?